src/main/java/com/github/rodionovsasha/commentservice/services/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.github.rodionovsasha.commentservice.services;
+
+import com.github.rodionovsasha.commentservice.entities.User;
+
+public interface UserService {
+    User create(String name, int age);
+
+    User getById(long id);
+
+    User getActiveUser(long userId);
+
+    void updateName(long id, String name);
+
+    void updateAge(long id, int age);
+
+    void deactivate(long id);
+
+    void activate(long id);
+
+    void delete(long id);
```

Comment:
Let's get rid of the `delete` method because we have `deactivate`, just to make it simpler